### DISCUSSION TITLE
Button submit proposal

### DISCUSSION THREAD

**Discussion by dbwebapp** [BitCannaGlobal/WeedWallet#80] [General]
The submit button is not working. 
Note that the Browser Shield are OFF.

![Screenshot 2023-05-25 at 08 46 52](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/e4948aee-990e-4ccf-8cae-5c7e73f5ea8c)
![Screenshot 2023-05-25 at 08 47 14](https://github.com/BitCannaGlobal/WeedWallet/assets/96944913/c014a3ce-e097-45b7-9e85-e0b6f39d7c7a)

**Comment by TJ-BitCanna** [comment]
Thanks! We'll look into it

(bold plans with a community pool spend btw? ;))

**Comment by TJ-BitCanna** [comment]
Suggested issue:
- make sure code is activated for the "Submit proposal" button

**Comment by TJ-BitCanna** [comment]
This page will be limited in terms of access. As a starting point only for the team, maybe in the future to more participants.

**Comment by TJ-BitCanna** [comment]
Page has been closed (for now) for the "regular" wallet-users.
